example_sns_init.yaml

```diff
@@ -0,0 +1,117 @@
+# You should make a copy of this file, name it sns_init.yaml, and edit it to
+# your taste.
+#
+# All principal IDs should almost certainly be changed.
+#
+# In this file, 1 year is nominally defined to be 365.25 days.
+#
+# This gets passed to `sns propose`. See propose_sns.sh.
+#
+# This follows the second format developed for the `sns` CLI. The first format
+# will be supported for a time, but this format will eventually become the
+# standard format.
```

Comment:
I got a bit confused what "formats" you are referring to here.

Reply:
Updated the wording here.